controllers/api/webhook.go

```diff
@@ -0,0 +1,117 @@
+package api
+
+//TODO
+
+import (
+  "net/http"
+  "strconv"
+  "encoding/json"
+
+  "github.com/gophish/gophish/models"
+  "github.com/gorilla/mux"
+  log "github.com/gophish/gophish/logger"
+  "github.com/gophish/gophish/webhook"
+)
+
```

Comment:
Please add a docstring here in the format `Webhooks ...`, such as `Webhooks handles the CRUD operations for managing Gophish webhooks` or similar.